cfpb/cfgov-refresh
# wellbeing: Support use without JavaScript, update content and vizualizations

Paired on most of the consequential bits of this with @chosak, and it's still behind a feature flag, so I'm going to merge straight away to try and get this to Build today so stakeholders can see a progress update.

## Removals

- Removes variants that were only needed for testing

## Changes

- Converts JS-based processing to Django back-end processing; project works well without JavaScript now
- Updates visualizations and content based on results of user testing

## Testing

1. Pull branch
2. Runserver
3. Activate `FWB_RELEASE` flag, if not already done
4. Visit http://localhost:8000/consumer-tools/financial-well-being/
5. Take the questionnaire, see the results

## Todos before Monday's release

- Move CSS and JS into standard Gulp workflow
  - Which will enable the use of the Analytics module to implement the desired custom events (see preview in main.js here)
- Write tests
- Convert radio buttons to CCT-like blue buttons
- Deal with Recent Posts module...
- A few minor content changes to come

## Checklist
